packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/java/auth-vs-oie.md

```diff
@@ -0,0 +1,18 @@
+
+For embedded Java applications deploying Okta Classic Engine authentication flows, there are generally two implementation approaches: using the Okta back-end APIs or using the [Okta Java Authentication (Auth) SDK](https://github.com/okta/okta-auth-java/). Some embedded apps used a combination of the two approaches to achieve their desired authentication flow. The Okta back-end API is seen as a manual way of implementing authentication flows with Okta. Whereas, the Java Auth SDK encapsulates the API resources to a convenient client wrapper for apps to achieve the same classic authentication flows.
+
+No matter which approach you used for your Classic Engine embedded app, to upgrade to an Identity Engine embedded app, you need to use an Identity Engine SDK. The upgrade process requires you to review your existing code and figure out the necessary mappings to transform API requests or Classic Engine SDK methods to Identity Engine SDK methods. In some instances, you will need to modify the interaction structure of your code.
+
+The following table is a high-level mapping of the typical Classic Engine API requests and Java Auth SDK methods to Java Identity Engine SDK methods. Since there are a variety of combinations to achieve Classic Engine authentication flows, the mapping list is not exhaustive.
```

Comment:
```suggestion
The following table is a high-level mapping of the typical Classic Engine API requests and Java Auth SDK methods to Java Identity Engine SDK methods. Since there are a variety of combinations to achieve Classic Engine authentication flows, the mapping list isn't exhaustive.
```